build-scripts/build-rpm.sh

```diff
@@ -13,9 +13,10 @@ rpmbuild \
     --define "_topmdir rpmbuild" \
     --define "_rpmdir rpmbuild" \
     --define "with_python ${WITH_PYTHON:=1}" \
+    --define "no_coverage ${NO_COVERAGE:=1}"\
     --rebuild rpmbuild/SRPMS/*src.rpm
 
 # Move RPMs to exported artifacts
-ARTIFACTS_DIR="${ARTIFACTS_DIR:=$(pwd)/artifacts/rpms-$(date +%04Y%02m%02d%02H%02M)}"
-[[ -d $ARTIFACTS_DIR ]] || mkdir -p $ARTIFACTS_DIR
-find rpmbuild -iname \*rpm | xargs mv -t $ARTIFACTS_DIR
+# ARTIFACTS_DIR="${ARTIFACTS_DIR:=$(pwd)/artifacts/rpms-$(date +%04Y%02m%02d%02H%02M)}"
+# [[ -d $ARTIFACTS_DIR ]] || mkdir -p $ARTIFACTS_DIR
+# find rpmbuild -iname \*rpm | xargs mv -t $ARTIFACTS_DIR
```

Comment:
What is this change for?

Reply:
Please revert, we need above commands to move artifacts into `$ARTIFACTS_DIR`, otherwise integration tests cannot install created RPMs